test/ffmpeg-qsv/encode/encoder.py

```diff
@@ -146,15 +140,14 @@ def check_output(self):
       assert m is not None, "It appears that the lookahead did not load"
 
   def check_metrics(self):
-    name = self.gen_name().format(**vars(self))
-    self.decoded = get_media()._test_artifact(
-      "{}-{width}x{height}-{format}.yuv".format(name, **vars(self)))
-
-    call(
-      "ffmpeg -hwaccel qsv -hwaccel_device /dev/dri/renderD128 -v verbose"
-      " -c:v {ffdecoder} -i {encoded} -vf 'hwdownload,format={hwformat}'"
-      " -pix_fmt {mformat} -f rawvideo -vsync passthrough -vframes {frames}"
-      " -y {decoded}".format(**vars(self)))
+    oopts = "-c:v {ffdecoder} -i {encoded}"
```

Comment:
assignment error, oopts should be changed to iopts and vice-versa

Reply:
Good catch.  I guess I only tested with jpeg encode (which I forgot overrides this method).

Fixed.